src/components/DataTable/DataTable.tsx

```diff
@@ -230,99 +158,48 @@ class DataTable extends React.PureComponent<CombinedProps, DataTableState> {
                 {totalsMarkup}
               </thead>
               <tbody>{bodyMarkup}</tbody>
-              {footerMarkup}
             </table>
           </div>
+          {footerMarkup}
         </div>
       </div>
     );
   }
 
-  private tallestCellHeights = () => {
```

Comment:
No more need to calculate/set/update row heights manually anymore. With the absolute positioning removed, normal table wrapping and resizing behavior is restored 🙌